include/orc/orc.hpp

```diff
@@ -54,7 +63,15 @@ void orc_reset();
 const char* demangle(const char* x);
 
 /**************************************************************************************************/
+// TODO: (fosterbrereton) Find a better home for this somewhere?
+template <class C>
+void sort_unique(C& container) {
```

Comment:
make_unique? sort_and_make_unique? Does 'sort' under state its function?

Reply:
The routine does two things. The first is that it sorts the collection of items (using `std::sort`). Then it scans those items and removes duplicates (using the `std::unique` algorithm. Unfortunately the name collides a bit with `make_unique`, which is completely different.)

Reply:
I was wondering if sort_unique was the right name - but I haven't come up with a better one, so it will do fine. :)